src/main/groovy/edu/wpi/first/gradlerio/wpi/WPIExtension.groovy

```diff
@@ -34,6 +34,9 @@ class WPIExtension {
 
     String frcYear = '2019'
 
+    boolean checkImage = true
```

Comment:
Is there a reason these shouldn't be in the Artifact?

Reply:
tbh I totally blanked that roborio was a modifyable easily class. fixing